packages/reactv2/src/components/Provider/index.tsx

```diff
@@ -18,22 +19,31 @@ export interface ProviderProps {
   navigate?: NavigateHandler
   theme?: Theme
   userId?: string
+  groupId?: string
 }
 
 interface ProviderContext extends Omit<ProviderProps, 'children' | 'theme'> {
   modals: string[]
   setModals: Dispatch<SetStateAction<string[]>>
+  setUserId: Dispatch<SetStateAction<string>>
+  setGroupId: Dispatch<SetStateAction<string>>
+  getConfig: () => FrigadeConfig
 }
 
 export const FrigadeContext = createContext<ProviderContext>({
   apiKey: '',
   modals: [],
   setModals: () => {},
   navigate: () => {},
+  setUserId: () => {},
+  setGroupId: () => {},
+  getConfig: () => ({}),
```

Comment:
We could prob structure the config internally as `config: FrigadeConfig` and say that the props are on the top level of the Provider for ease of use, but are grouped as `config` in context for... also ease of use but different.